pkg/webui/components/link/index.js

```diff
@@ -163,9 +166,9 @@ const DocLink = function(props) {
       name={name}
       onClick={onClick}
     >
-      <Icon className={style.docIcon} icon="book" />
+      {beforeIcon && <Icon className={style.docIcon} icon={beforeIcon} />}
       {children}
-      <Icon className={style.icon} icon="launch" />
+      {afterIcon && <Icon className={style.icon} icon={afterIcon} />}
```

Comment:
<img width="214" alt="Screenshot 2020-09-09 at 09 51 55" src="https://user-images.githubusercontent.com/16374166/92564641-29d95280-f282-11ea-92ce-4e7e7f30159e.png">
Doesnt look right

Reply:
this is from `<DocLink />`, I didn't change the styling. How should it look?

Reply:
First you have to remove left margin from the icon element and add it to the link instead. Then adjust link styles so it takes width and height required for the icon.
<img width="603" alt="Screenshot 2020-09-23 at 18 06 11" src="https://user-images.githubusercontent.com/16374166/94031625-c09c2800-fdc7-11ea-8f99-8cd395d1e7e6.png">

vs

<img width="603" alt="Screenshot 2020-09-23 at 18 07 05" src="https://user-images.githubusercontent.com/16374166/94031623-bf6afb00-fdc7-11ea-8b97-6e9c736c7745.png">

We might even remove the default outline of links in this case and the icon color on focus. cc @kschiffer

Reply:
Yeah this needs some love. I think the outline is fine here as long as the bounding box is correct. I'll push some fixes here myself.